api/BaseAudioContext.json

```diff
@@ -178,15 +169,20 @@
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/BaseAudioContext/createBiquadFilter",
           "support": {
-            "chrome": {
-              "version_added": "10",
-              "prefix": "webkit"
-            },
+            "chrome": [
+              {
+                "version_added": "56"
+              },
+              {
+                "version_added": "14",
+                "prefix": "webkit"
```

Comment:
I think this is also wrong, that there's been no `webkitCreateBiquadFilter`. There are a bunch more of these that follow, so I won't comment on each one.